Sources/Shadow/Shadow.swift

```diff
@@ -117,6 +117,15 @@ private struct ShadowViewModifier: ViewModifier {
                     radius: isPressed ? shadow.radius2 * shadow.pressedModifier : shadow.radius2,
                     x: shadow.x2,
                     y: isPressed ? shadow.y2 * shadow.pressedModifier : shadow.y2)
+            .onLongPressGesture(
+                minimumDuration: 0,
+                pressing: { inProgress in
+                    isPressed = inProgress
+                },
+                perform: {
+                    isPressed = false
+                }
+            )
```

Comment:
I don't think this will work well as gesture interaction is dependent on order of the modifiers, so this can break tap gestures in the views that adopt the shadow modifier. Might be able to get around it with a combination of `highPriorityGesture` and `simultaneousGesture`, but that won't help for views with built in gestures such as `Button`. I don't really see the use case for this pressed shadow state anywhere else than a `Button`, so if it's one thing we don't want to break it's that 😄